queries.sql

```diff
@@ -8,3 +8,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE (weight_kg >= 10.4) AND (weight_kg <= 17.3);
+
+/* Transaction */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'digimon' WHERE name IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Please make sure to carefully read the requirements before writing a query as you don't have to do a rollback after committing the changes at line `24`.